package.json

```diff
@@ -1,234 +1,234 @@
-      },
-      {
-        "command": "musictime.showAlbum",
-        "title": "Show album"
-      }
-    ],
-    "viewsContainers": {
-      "activitybar": [
-        {
-          "id": "music-time-sidebar",
-          "title": "Music Time",
-          "icon": "resources/dark/headphone-symbol.svg"
-        }
-      ]
-    },
-    "views": {
-      "music-time-sidebar": [
-        {
-          "id": "musictime.webView",
-          "type": "webview",
-          "name": "",
-          "icon": "resources/dark/headphone-symbol.svg"
-        }
-      ]
-    }
-  },
-  "devDependencies": {
-    "@eslint/js": "^9.3.0",
-    "@types/copy-webpack-plugin": "^8.0.1",
-    "@types/mocha": "^9.0.0",
-    "@types/node": "^16.10.3",
-    "@types/vscode": "^1.52.0",
-    "@typescript-eslint/eslint-plugin": "^4.33.0",
-    "@typescript-eslint/parser": "^4.33.0",
-    "copy-webpack-plugin": "^9.0.1",
-    "eslint": "9.x",
-    "file-loader": "^6.2.0",
-    "globals": "^15.3.0",
-    "ts-loader": "^9.2.6",
-    "tslint": "^6.1.3",
-    "tslint-config-prettier": "^1.18.0",
-    "typescript": "^4.5.5",
-    "typescript-eslint": "^7.11.0",
-    "webpack": "^5.76.0",
-    "webpack-cli": "^4.9.2"
-  },
-  "dependencies": {
-    "@slack/web-api": "^7.0.4",
-    "@types/uuid": "^9.0.8",
-    "@types/ws": "^8.5.10",
-    "axios": "^1.7.2",
-    "clipboardy": "^4.0.0",
-    "cody-music": "2.10.7",
-    "date-fns": "^3.6.0",
-    "open": "^8.4.0",
-    "query-string": "^7.0.1",
-    "uuid": "^8.3.2",
-    "ws": "^8.11.0"
-  }
+	"name": "music-time",
+	"displayName": "Music Time for Spotify",
+	"version": "2.2.45",
+	"publisher": "softwaredotcom",
+	"description": "Music Time for Spotify is a VS Code extension that discovers the most productive music to listen to as you code.",
+	"author": {
+		"name": "Software.com"
+	},
+	"license": "SEE LICENSE IN LICENSE",
+	"icon": "images/app-icon-music-time.png",
+	"galleryBanner": {
+		"color": "#384356",
+		"theme": "dark"
+	},
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/swdotcom/swdc-vscode-musictime"
+	},
+	"extensionKind": [
+		"ui",
+		"workspace"
+	],
+	"categories": [
+		"Other"
+	],
+	"keywords": [
+		"spotify",
+		"music",
+		"cody",
+		"playlists",
+		"tracks",
+		"productivity",
+		"songs",
+		"player",
+		"time"
+	],
+	"activationEvents": [
+		"onStartupFinished"
+	],
+	"engines": {
+		"vscode": "^1.52.0"
+	},
+	"main": "./dist/extension",
+	"scripts": {
+		"lint": "tslint -p ./",
+		"build": "vsce package --yarn",
+		"watch": "tsc -watch -p ./",
+		"webpack": "yarn compile:extension & yarn compile:views",
+		"webpack-dev": "yarn watch:extension & yarn watch:views",
+		"compile:extension": "tsc -p ./",
+		"compile:views": "webpack --mode development",
+		"watch:extension": "tsc -watch -p ./",
+		"watch:views": "webpack --watch --mode development",
+		"vscode:prepublish": "webpack --mode production",
+		"test-compile": "tsc -p ./"
+	},
+	"contributes": {
+		"commands": [
+			{
```

Comment:
the whitespace is a bit much

Reply:
yeah, not sure what happened. used prettier to reformat